test/system_SUITE.erl

```diff
@@ -687,7 +687,7 @@ queue_unbind(Config) ->
 
 %% -------------------------------------------------------------------
 
-%% This is designed to exercize the internal queuing mechanism
+%% This is designed to exercise the internal queuing mechanism
```

Comment:
rabbiqmq is en-GB, -ize is a en-US ending, but even so, this is still not correct in en-US...

Reply:
Sticking to the UK spelling is becoming increasingly harder as the majority of our team are based outside of the UK now. We don't feel particularly strongly either way :)

Reply:
Yeah, I'm familiar w/ all sides of the problem.

If you're feeling adventurous, you could try adding a Travis hook to improve/retain spelling behaviors (or behaviours, up to you...). I've provided a hook for two projects, one has held the line w/ it and seems happy, one ran away screaming after a week or so.